spec/helpers/users_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
remove this file?

Reply:
removed

Reply:
when you are 100% sure that the requirement has been done you can "Resolve conversation"